full_install.sh

```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env bash
```

Comment:
Looks good! As a question, any ideas on making this dev setup process more flexible? i.e., perhaps having a root level setup.py that installs all the submodules so someone can do:

```
git clone https://github.com/awslabs/autogluon
cd autogluon && python3 setup.py develop
```

This was the code required in v0.0.15 prior to namespace modules, but doesn't work anymore since there isn't a root setup.py

Reply:
if something comes to mind I will let you know. You can include urls in setup.py so maybe it would work to use local paths to the submodules from a parent setup.py as you describe.